hocs/withAuthorization.tsx

```diff
@@ -28,9 +28,11 @@ const withAuthorization = <Props extends object>( //eslint-disable-line
   Page: React.ComponentType<Props & { editable: boolean }>,
 ): React.ComponentType<Props> =>
   function PageWithAuthorization(props) {
-    const { user } = useAuth();
+    const { user, authStateChangeFinished } = useAuth();
     const id = useRouter().query.id as string;
 
+    if (!authStateChangeFinished) return <></>;
```

Comment:
why not null?

Reply:
right, returning `null` is a better convention, isn't it? Or is it actually better?
Anyways, I changed it to return `null` now.

Reply:
haha. Sorry. Thought about it last night, and forgot to comment: I think we should show the loading spinner here (after all, we're _waiting_ for the auth change to finish). Wouldn't you agree?

Reply:
ah okay haha. Yes, sure. We can add the loading spinner here.